Makefile

```diff
@@ -453,14 +453,6 @@ create-workload-cluster: $(ENVSUBST)
 	kubectl get secrets $(CLUSTER_NAME)-kubeconfig -o json | jq -r .data.value | base64 --decode > ./kubeconfig
 	timeout --foreground 600 bash -c "while ! kubectl --kubeconfig=./kubeconfig get nodes | grep master; do sleep 1; done"
 
-	# Deploy calico
-	@if [[ "${CLUSTER_TEMPLATE}" == *ipv6* ]]; then \
-		kubectl --kubeconfig=./kubeconfig apply -f templates/addons/calico-ipv6.yaml; \
-	else \
-		kubectl --kubeconfig=./kubeconfig apply -f templates/addons/calico.yaml; \
```

Comment:
// TODO: check if conformance templates have the CNI label

Reply:
so the issue with this is that the conformance tests have the "wrong" CNI label, they e2e tests have one ClusterResourceSet per cluster whereas the makefile target has one common ClusterResourceSet